docker-compose.yml

```
@@ -38,6 +38,8 @@ services:
       - BACKEND_TOKEN_KEY=${BACKEND_TOKEN_KEY}
       - BACKEND_TOKEN_USER=${BACKEND_TOKEN_USER}
       - BACKEND_TOKEN_PASSWORD=${BACKEND_TOKEN_PASSWORD}
+      - BACKEND_TMP_USAGE_DURATION=${BACKEND_TMP_USAGE_DURATION}
+      - BACKEND_TMP_USAGE_WINDOW=${BACKEND_TMP_USAGE_WINDOW}
```

Everywhere else is called `BACKEND_TMP_DURATION` not `BACKEND_TMP_USAGE_DURATION` 
```suggestion
      - BACKEND_TMP_DURATION=${BACKEND_TMP_DURATION}
      - BACKEND_TMP_WINDOW=${BACKEND_TMP_WINDOW}
```